Package.swift

```diff
@@ -248,7 +248,7 @@ let package = Package(
 if ProcessInfo.processInfo.environment["SWIFTPM_LLBUILD_FWK"] == nil {
     if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         package.dependencies += [
-            .package(url: "https://github.com/apple/swift-llbuild.git", .branch("master")),
+            .package(url: "https://github.com/apple/swift-llbuild.git", .branch("main")),
```

Comment:
I think these have already been addressed in a separate PR, but I could be wrong.  In any case they should merge cleanly if this is rebased on it.

Reply:
Yep it has been addressed in a separate PR #2951. I cherry-picked 44685b3fa4612927e42d4275db5ad0fbba09fbbe just so I could run tests after the rebranch. Should I revert the cherry-pick and rebase my branch onto main instead?

Reply:
I just rebased my branch onto the main branch, because there are more problems from the rebranching that stopped me from building the project. However, I think I messed it up. Now there are 42 commits and 36 files changed in this PR, and I dragged the authors of all those commits into this PR as participants (Sorry!). What can I do to remedy this? Should I create a new branch from the main branch, and move my commits there, then open a new PR?

Reply:
I did `git reset HEAD~1`, and force-pushed. Everything should be fine now. I'm going to mark this one as resolved, because this PR's branch is now rebased on the main branch, which has merged the PR that addressed swift-llbuild's rebranching.